README.md

```diff
@@ -187,7 +187,7 @@ directive in the OPAL such that only the data really needed is downloaded.
 * `url` defines the observe url to query the data from.
 
 
-* `user` defines the user account to use for the export of the data.
+* `user` is the first part of the url before `observeinc.com` (this parameter is optional, since it is infered usually). This is not the ID of your personal account. You don't have to specify that since you use an access token. 
```

Comment:
"user specifies the customer id, i.e., the first part of the url before `observeinc.com`; as this is part of the url already, specifying it is optional"

This change also need to be included in the observe_export.py, as the readme here is auto-generated from the code

Reply:
Done. How did you generate the readme from code?

Reply:
As always, see readme: 

> The following documentation is automatically generated from the source using sphinx. You can create a HTML documentation by calling make html in the docs directory.